src/gha_logs.rs

```diff
@@ -189,11 +225,18 @@ async fn process_logs(
             `<span class="warning-marker">##[warning]</span>`
         );
 
-        // 5. Add the html to the DOM
+        // 5. Add anchors to GitHub around some paths
+        const pathRegex = /(?<boundary>[^a-zA-Z0-9.\\/])(?<inner>(?:[\\\/]?(?:checkout[\\\/])?(?<path>(?:{tree_roots})[\\\/][a-zA-Z0-9_$\-.\\\/]+))(?::(?<line>[0-9]+):(?<col>[0-9]+))?)/g;
```

Comment:
Could you please add a comment here that shows some examples of paths that are (and aren't) parsed? And maybe also explain the `tree_roots` "magic" a bit.

Reply:
Sure, I can add some comments.

Regarding the examples, would it be fine if I just redirected to https://regex101.com/r/vCnx9Y/2 ? I think it would be much easier to understand the regex.

Reply:
Sure, done.